strictdoc/export/excel/excel_generator.py

```diff
@@ -1,59 +1,193 @@
 import os
 from pathlib import Path
+from typing import Dict, List
 
 import xlsxwriter
+from xlsxwriter.workbook import Workbook
+from xlsxwriter.worksheet import Worksheet
+from xlsxwriter.utility import xl_cell_to_rowcol_abs
 
 from strictdoc.backend.dsl.models.document import Document
+from strictdoc.backend.dsl.models.requirement import Requirement
 from strictdoc.core.document_iterator import DocumentCachingIterator
 from strictdoc.core.document_tree import DocumentTree
 
 
-def get_path_components(folder_path):
-    path = os.path.normpath(folder_path)
-    return path.split(os.sep)
+EXCEL_SHEET_NAME = "requierements"
+MAX_WIDTH = 75
+HEADER_MARGIN = 3
+MAX_WIDTH_KEY = "max_width"
+COLUMN_HEADER_KEY = "header"
+PARENT_COLUMN_HEADER_LABEL = "PARENT"
+EXPORT_COLUMNS = [
+    {
+        "name": "uid",
+        "header": "UID",
+    },
+    # {
+    #     "name": "title",
+    #     "header": "TITLE",
+    # },
+    {
+        "name": "statement",
+        "header": "STATEMENT",
+    },
+    # {
+    #     "name": "rationale",
+    #     "header": "RATIONALE",
+    # },
+    # {
+    #     "name": "body",
+    #     "header": "BODY",
+    # },
+]
 
 
 class ExcelGenerator:
+    @staticmethod
+    def lookup_refs(document_contents: List) -> Dict[str, int]:
+        """Lookups each req position in the list, keeping track of where it will be placed in the excel sheet
```

Comment:
I know it is a good practice for Python projects to keep the comments, but I haven't been using them until now. This might be something that I will start doing but until then, let's have all code to be consistent.
Please remove all of the `"""` comments.

Reply:
are you certain? what is the idea behind it? selection of the comment format? google vs numpy vs,... ?

Reply:
I am fairly certain at this point. I have worked with Doorstop for a quite a while, and it was heavily documented. I found that I never looked at the comments, only at the code. I might reconsider this later when I am ready for a general approach when comments have to be everywhere, including automatic enforcement with software. Until then, I think the existing code is fairly simple and self-evident at this point.